src/cmd.zig

```diff
@@ -2049,6 +2102,7 @@ const ReplayAndConsensusServiceState = struct {
             replay_threads: u32,
             disable_consensus: bool,
             voting_enabled: bool,
+            vote_account_pubkey: ?Pubkey,
```

Comment:
the _pubkey suffix seems somewhat redundant in context, no?

Reply:
No strong feelings, but there exist the usage of this suffix across the codebase

Reply:
Went with `vote_account_address` [f032fa7](https://github.com/Syndica/sig/pull/1088/commits/f032fa7870ff1d196182670deb3f4fe9caadf1d0)